### ISSUE TITLE
6.2.0 Rolling Release

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/ocis#9706]
### Prerequisites

* [x] DEV: release reva v.2.20.0 https://github.com/cs3org/reva/pull/4781
* [x] DEV: bump reva to v.2.20.0 https://github.com/owncloud/ocis/pull/9708
* [x] DEV: bump web to v.10.0.0 https://github.com/owncloud/ocis/pull/9707
* [x] DEV: Create tag `v6.2.0-rc.1`
* [x] DEV: Create signed tag  
* [x] DEV: Check successful CI run on  tag / BLOCKING for all further activity

## QA Phase
* [x] QA: Performance test https://grafana.k6.infra.owncloud.works/?orgId=1&refresh=15m&var-test_id=5da732e8-6e80-4191-b3dd-86e06d91586a
* [x] QA: QA: Compatibility testing between `owncloud/ocis-rolling:6.1.0 `and `owncloud/ocis-rolling:master(similar with 6.2.0)`
* [x] QA: e2e with different deployment: https://github.com/owncloud/web/pull/11297

### After QA Phase
* [x] DEV: Create branch `release-x.x.x`
  * [x] DEV: bump ocis version in necessary files
  * [x] DEV: `ocis-pkg/version/version.go`
  * [x] DEV: `sonar-project.properties`
  * [x] DEV: released deployment versions
  * [x] DEV: prepare changelog folder in `changelog/x.x.x_???`
* [x] DEV: Create final signed tag
* [x] DEV: Check successful CI run on `vx.y.z` tag / BLOCKING for all further activity
* [x] Merge release notes https://github.com/owncloud/docs-main/pull/64
* [x] roll out release on https://infinite.owncloud.com/
* [ ] roll out rolling release on https://owncloud.dev/ocis/deployment/continuous_deployment/ https://github.com/owncloud/ocis/pull/9717

****

**Comment by ScharfViktor** [comment]
### Compatibility test between `owncloud/ocis:6.0.0` and `owncloud/ocis-rolling:6.1.0-alpha.2`

shares:
- [x] public share (with an expiration date)
- [x] share with user as role editor (with an expiration date)
- [x] share with user as role viewer
- [x] share with group "Users"

project space:
- [x] uploaded folder & files
- [x] trashed files
- [x] share space with user
- [x] public share space

files:
- [x] uploaded folder & files
- [x] version of files
- [x] trashed files
- [x] GDPR export

users
- [x] non demo user with role admin
- [x] non demo user with role user
- [x] created new user
- [x] disabled user
- [x] created new group

settings
- [x] language settings
- [x] update logo админ
- [x] new password


Remarks: 

- user cannot open file in Collabora

@micbar probably related to https://github.com/owncloud/ocis/pull/9366 ?
log: 

```
2024-07-30 10:08:36 wsd-00001-00022 2024-07-30 08:08:36.997791 +0000 [ websrv_poll ] ERR  #37: SSL verification warning (handshake): self-signed certificate| net/SslSocket.hpp:448
2024-07-30 10:08:36 wsd-00001-00022 2024-07-30 08:08:36.997927 +0000 [ websrv_poll ] ERR  #37: Error while handling poll at 0 in websrv_poll: #37 (handshake): unknown. BIO error: 369098857, rc: -1: error:16000069:STORE routines::unregistered scheme:
2024-07-30 10:08:36 40D061A9FFFF0000:error:16000069:STORE routines:ossl_store_get0_loader_int:unregistered scheme:crypto/store/store_register.c:237:scheme=file
2024-07-30 10:08:36 
2024-07-30 10:08:36 40D061A9FFFF0000:error:80000002:system library:file_open:No such file or directory:providers/implementations/storemgmt/file_store.c:267:calling stat(/usr/local/ssl/certs)
2024-07-30 10:08:36 
2024-07-30 10:08:36 40D061A9FFFF0000:error:16000069:STORE routines:ossl_store_get0_loader_int:unregistered scheme:crypto/store/store_register.c:237:scheme=file
2024-07-30 10:08:36 
2024-07-30 10:08:36 40D061A9FFFF0000:error:80000002:system library:file_open:No such file or directory:providers/implementations/storemgmt/file_store.c:267:calling stat(/usr/local/ssl/certs)
2024-07-30 10:08:36 
2024-07-30 10:08:36 40D061A9FFFF0000:error:0A000086:SSL routines:tls_post_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1889:
2024-07-30 10:08:36 | net/Socket.cpp:564
2024-07-30 10:08:36 WOPI::CheckFileInfo failed for URI [https://wopiserver.owncloud.test/wopi/files/f1c275aaa43dbad4e65a1be1711709f4d256251a30d23bca23c9808b0b9b6718?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJXb3BpQ29udGV4dCI6eyJBY2Nlc3NUb2tlbiI6ImZBdHdoM1k1cDBKUUc5QnRRMkRlbEhWbWxQWXI4RHBTSG81NVZGV3hkeW4xYUhnRnJoNDd1d2NYTVBGOTFidVI0YW5IQUc5ZHlwVTNkVHAwVkttYTVqRkpsNzdmeWstNTBWcFcwR1VyQTlCZVRoN2xQRWxNNzJyUXBZUjRxR1NEZzVBTlZvazZ0dlAxRmlDTlhhN1JRVndvQXZSUTB0b1JWNlBsOWlLa0FKNktaOUFBaGV1SjUwSnhvOUNDRDU0VlhvZXVrUFpQUVFQSDRadGJEV1plYlZ3X2lvTmFuWXZIOVJnVXdRd2pHZHRwakZzOUtRaFdFRXFSamhBV01SSTdwWmFScXdrUmtqd2xxQU5GNXhmc09aN2xoV01sbFR2VEt5dW9RNFVEeEtOX0pGWTRrMkRIX2ZKOExTZm5QU1ZzQUsxODF2NUp5ZTd6NUhtbmlVUTNuUGxtendPNFN6U0pEaVdHOTdsbUd0R1hFR0NsYllGQW0wYVFjVHp5WlFaYWlZamJsTE9wZlBWd3V1eFl2d09JRHVKbTNFTXZZTFlIclQ4MUFiVEJhZE1wU0lYSEZkRTJhaXB1dnJIRkhhRzB2aVhlUnpUVGc0ZUlVazVTMFBfOFBqVFJoaFZvR0c3SDV1bGxsOFdlLWpwamxjQ0ttS2U4cjNQM2M2YWJ0SFBTckxFd2Fnai02MDl3bGoxOWdfQnB2Y1lhR3Y5NlNQcURleUVUMHdKajI5ZEg5OVUxRm01Q0NBMElhNG54ODEtRFJnQXFyV2t2U2UwbzdDbElHZzlCMXJQQTB5ai1VQXo4a3AxakpDYWlhc3NFWjhaZmsxdl9pLXZjQ2hEVFB5WnhuTFlTWC1MbnhPOVd0ZmxQRjdlZjlJWWthaEJzQ0tWbHh0dDRZWXFVSHVNTUVmTFdOdjdaWXNvNHA1TTQtN0p1SFJVYkhhUkZjZ0xfbDRMWjFpXzNjeDhKYjZyNUF2Z0NSZ3BvcW50U1l3X1kxTTBWMk5vdnNoR1VMQzB1U1BPTXBNYklYLWV4aFNNUmtTVWlhZzFjdThiMElISXRvSkVMT0xxMWFnPT0iLCJWaWV3T25seVRva2VuIjoiIiwiRmlsZVJlZmVyZW5jZSI6eyJyZXNvdXJjZV9pZCI6eyJzdG9yYWdlX2lkIjoiY2Y3MTVjNzQtNmVkYS00YjA3LWIyNDYtZDM2YTJlODU4MTk0Iiwib3BhcXVlX2lkIjoiN2QxYWYwYjAtMTQ2OS00OWIzLWE2ZmItZjZmNjJmMTgwM2Y2Iiwic3BhY2VfaWQiOiJiOWY3NmYzNi02MDg2LTQ3NmMtOTJkMi05YThlNTViM2U4NGQifSwicGF0aCI6Ii4ifSwiVXNlciI6eyJpZCI6eyJpZHAiOiJodHRwczovL29jaXMub3duY2xvdWQudGVzdCIsIm9wYXF1ZV9pZCI6ImI5Zjc2ZjM2LTYwODYtNDc2Yy05MmQyLTlhOGU1NWIzZTg0ZCIsInR5cGUiOjF9LCJ1c2VybmFtZSI6ImFkbWluIiwibWFpbCI6ImFkbWluQGV4YW1wbGUub3JnIiwiZGlzcGxheV9uYW1lIjoiQWRtaW4iLCJ1aWRfbnVtYmVyIjo5OSwiZ2lkX251bWJlciI6OTl9LCJWaWV3TW9kZSI6MywiRWRpdEFwcFVybCI6Imh0dHBzOi8vY29sbGFib3JhLm93bmNsb3VkLnRlc3QvYnJvd3Nlci80OGJmMGJmL2Nvb2wuaHRtbD9VSV9MTENDPXJ1XHUwMDI2V09QSVNyYz1odHRwcyUzQSUyRiUyRndvcGlzZXJ2ZXIub3duY2xvdWQudGVzdCUyRndvcGklMkZmaWxlcyUyRmYxYzI3NWFhYTQzZGJhZDRlNjVhMWJlMTcxMTcwOWY0ZDI1NjI1MWEzMGQyM2JjYTIzYzk4MDhiMGI5YjY3MThcdTAwMjZsYW5nPXJ1XHUwMDI2dWk9cnUiLCJWaWV3QXBwVXJsIjoiaHR0cHM6Ly9jb2xsYWJvcmEub3duY2xvdWQudGVzdC9icm93c2VyLzQ4YmYwYmYvY29vbC5odG1sP1VJX0xMQ0M9cnVcdTAwMjZXT1BJU3JjPWh0dHBzJTNBJTJGJTJGd29waXNlcnZlci5vd25jbG91ZC50ZXN0JTJGd29waSUyRmZpbGVzJTJGZjFjMjc1YWFhNDNkYmFkNGU2NWExYmUxNzExNzA5ZjRkMjU2MjUxYTMwZDIzYmNhMjNjOTgwOGIwYjliNjcxOFx1MDAyNmxhbmc9cnVcdTAwMjZ1aT1ydSJ9LCJleHAiOjE3MjI0MTMzMTZ9.SKbM6Y6fIqlyoq9VuwzS1xAojXS6MNWWGUXMZrzwXto&access_token_ttl=1722413316000&permission=edit]: 0 (Unknown) . Headers:  Body: []| wsd/wopi/CheckFileInfo.cpp:95
2024-07-30 10:08:36 wsd-00001-00022 2024-07-30 08:08:36.998029 +0000 [ websrv_poll ] ERR  #37: Invalid URI or access denied to [https://wopiserver.owncloud.test/wopi/files/f1c275aaa43dbad4e65a1be1711709f4d256251a30d23bca23c9808b0b9b6718?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJXb3BpQ29udGV4dCI6eyJBY2Nlc3NUb2tlbiI6ImZBdHdoM1k1cDBKUUc5QnRRMkRlbEhWbWxQWXI4RHBTSG81NVZGV3hkeW4xYUhnRnJoNDd1d2NYTVBGOTFidVI0YW5IQUc5ZHlwVTNkVHAwVkttYTVqRkpsNzdmeWstNTBWcFcwR1VyQTlCZVRoN2xQRWxNNzJyUXBZUjRxR1NEZzVBTlZvazZ0dlAxRmlDTlhhN1JRVndvQXZSUTB0b1JWNlBsOWlLa0FKNktaOUFBaGV1SjUwSnhvOUNDRDU0VlhvZXVrUFpQUVFQSDRadGJEV1plYlZ3X2lvTmFuWXZIOVJnVXdRd2pHZHRwakZzOUtRaFdFRXFSamhBV01SSTdwWmFScXdrUmtqd2xxQU5GNXhmc09aN2xoV01sbFR2VEt5dW9RNFVEeEtOX0pGWTRrMkRIX2ZKOExTZm5QU1ZzQUsxODF2NUp5ZTd6NUhtbmlVUTNuUGxtendPNFN6U0pEaVdHOTdsbUd0R1hFR0NsYllGQW0wYVFjVHp5WlFaYWlZamJsTE9wZlBWd3V1eFl2d09JRHVKbTNFTXZZTFlIclQ4MUFiVEJhZE1wU0lYSEZkRTJhaXB1dnJIRkhhRzB2aVhlUnpUVGc0ZUlVazVTMFBfOFBqVFJoaFZvR0c3SDV1bGxsOFdlLWpwamxjQ0ttS2U4cjNQM2M2YWJ0SFBTckxFd2Fnai02MDl3bGoxOWdfQnB2Y1lhR3Y5NlNQcURleUVUMHdKajI5ZEg5OVUxRm01Q0NBMElhNG54ODEtRFJnQXFyV2t2U2UwbzdDbElHZzlCMXJQQTB5ai1VQXo4a3AxakpDYWlhc3NFWjhaZmsxdl9pLXZjQ2hEVFB5WnhuTFlTWC1MbnhPOVd0ZmxQRjdlZjlJWWthaEJzQ0tWbHh0dDRZWXFVSHVNTUVmTFdOdjdaWXNvNHA1TTQtN0p1SFJVYkhhUkZjZ0xfbDRMWjFpXzNjeDhKYjZyNUF2Z0NSZ3BvcW50U1l3X1kxTTBWMk5vdnNoR1VMQzB1U1BPTXBNYklYLWV4aFNNUmtTVWlhZzFjdThiMElISXRvSkVMT0xxMWFnPT0iLCJWaWV3T25seVRva2VuIjoiIiwiRmlsZVJlZmVyZW5jZSI6eyJyZXNvdXJjZV9pZCI6eyJzdG9yYWdlX2lkIjoiY2Y3MTVjNzQtNmVkYS00YjA3LWIyNDYtZDM2YTJlODU4MTk0Iiwib3BhcXVlX2lkIjoiN2QxYWYwYjAtMTQ2OS00OWIzLWE2ZmItZjZmNjJmMTgwM2Y2Iiwic3BhY2VfaWQiOiJiOWY3NmYzNi02MDg2LTQ3NmMtOTJkMi05YThlNTViM2U4NGQifSwicGF0aCI6Ii4ifSwiVXNlciI6eyJpZCI6eyJpZHAiOiJodHRwczovL29jaXMub3duY2xvdWQudGVzdCIsIm9wYXF1ZV9pZCI6ImI5Zjc2ZjM2LTYwODYtNDc2Yy05MmQyLTlhOGU1NWIzZTg0ZCIsInR5cGUiOjF9LCJ1c2VybmFtZSI6ImFkbWluIiwibWFpbCI6ImFkbWluQGV4YW1wbGUub3JnIiwiZGlzcGxheV9uYW1lIjoiQWRtaW4iLCJ1aWRfbnVtYmVyIjo5OSwiZ2lkX251bWJlciI6OTl9LCJWaWV3TW9kZSI6MywiRWRpdEFwcFVybCI6Imh0dHBzOi8vY29sbGFib3JhLm93bmNsb3VkLnRlc3QvYnJvd3Nlci80OGJmMGJmL2Nvb2wuaHRtbD9VSV9MTENDPXJ1XHUwMDI2V09QSVNyYz1odHRwcyUzQSUyRiUyRndvcGlzZXJ2ZXIub3duY2xvdWQudGVzdCUyRndvcGklMkZmaWxlcyUyRmYxYzI3NWFhYTQzZGJhZDRlNjVhMWJlMTcxMTcwOWY0ZDI1NjI1MWEzMGQyM2JjYTIzYzk4MDhiMGI5YjY3MThcdTAwMjZsYW5nPXJ1XHUwMDI2dWk9cnUiLCJWaWV3QXBwVXJsIjoiaHR0cHM6Ly9jb2xsYWJvcmEub3duY2xvdWQudGVzdC9icm93c2VyLzQ4YmYwYmYvY29vbC5odG1sP1VJX0xMQ0M9cnVcdTAwMjZXT1BJU3JjPWh0dHBzJTNBJTJGJTJGd29waXNlcnZlci5vd25jbG91ZC50ZXN0JTJGd29waSUyRmZpbGVzJTJGZjFjMjc1YWFhNDNkYmFkNGU2NWExYmUxNzExNzA5ZjRkMjU2MjUxYTMwZDIzYmNhMjNjOTgwOGIwYjliNjcxOFx1MDAyNmxhbmc9cnVcdTAwMjZ1aT1ydSJ9LCJleHAiOjE3MjI0MTMzMTZ9.SKbM6Y6fIqlyoq9VuwzS1xAojXS6MNWWGUXMZrzwXto&access_token_ttl=1722413316000&permission=edit]| wsd/wopi/CheckFileInfo.cpp:109
```

**Comment by micbar** [comment]
@ScharfViktor 
The issue with the local domain on collabora is known, not happening on a remote setup https://ocis.ocis.master.owncloud.works

**Comment by ScharfViktor** [comment]
> @ScharfViktor The issue with the local domain on collabora is known, not happening on a remote setup https://ocis.ocis.master.owncloud.works

yeah, I checked https://ocis.ocis.master.owncloud.works -> works fine with code:24.04.5.1.1 collabora version

that means we are done with QA phase

**Comment by micbar** [comment]
# 🎉  Infinite Scale v6.2.0

## Docker

`docker pull owncloud/ocis-rolling:latest`
`docker pull owncloud/ocis-rolling:6.2.0`

## Binaries

https://download.owncloud.com/ocis/ocis/rolling/6.2.0/

## Changelog

### oCIS

* Bugfix - Fix restarting of postprocessing: [#6945](https://github.com/owncloud/ocis/pull/6945)
* Bugfix - Fix crash on empty tracing provider: [#9622](https://github.com/owncloud/ocis/pull/9622)
* Bugfix - Fixed the file name validation if nodeid is used: [#9634](https://github.com/owncloud/ocis/pull/9634)
* Bugfix - Fix a missing SecureView permission attribute in the REPORT response: [#9638](https://github.com/owncloud/ocis/pull/9638)
* Bugfix - Fixed the channel lock in a workers pool: [#9647](https://github.com/owncloud/ocis/pull/9647)
* Bugfix - Missing invitation in permission responses: [#9652](https://github.com/owncloud/ocis/pull/9652)
* Bugfix - Repair nats-js-kv registry: [#9662](https://github.com/owncloud/ocis/pull/9662)
* Bugfix - Fix panic: [#9673](https://github.com/owncloud/ocis/pull/9673)
* Bugfix - Fixed the response code when copying the shared from to personal: [#9677](https://github.com/owncloud/ocis/pull/9677)
* Bugfix - Fixed response code for DELETE file that is in postprocessing: [#9689](https://github.com/owncloud/ocis/pull/9689)
* Change - Remove unavailable web config options: [#9679](https://github.com/owncloud/ocis/pull/9679)
* Enhancement - Introduce auth-app service: [#9079](https://github.com/owncloud/ocis/pull/9079)
* Enhancement - Add support for proof keys for the collaboration service: [#9366](https://github.com/owncloud/ocis/pull/9366)
* Enhancement - Log user agent and remote addr on auth errors: [#9475](https://github.com/owncloud/ocis/pull/9475)
* Enhancement - Add missing WOPI features: [#9580](https://github.com/owncloud/ocis/pull/9580)
* Enhancement - Bump commitID for web: [#9631](https://github.com/owncloud/ocis/pull/9631)
* Enhancement - Remove oidc-go dependency: [#9641](https://github.com/owncloud/ocis/pull/9641)
* Enhancement - Improve the collaboration service logging: [#9653](https://github.com/owncloud/ocis/pull/9653)
* Enhancement - Fix trash command: [#9665](https://github.com/owncloud/ocis/pull/9665)
* Enhancement - Added the debugging to full ocis docker example: [#9666](https://github.com/owncloud/ocis/pull/9666)
* Enhancement - Add locking support for MS Office Online Server: [#9685](https://github.com/owncloud/ocis/pull/9685)
* Enhancement - Add `--diff` to the `ocis init` command: [#9693](https://github.com/owncloud/ocis/pull/9693)

### Web

- Bugfix [owncloud/web#11174](https://github.com/owncloud/web/pull/11174): Downloading files via the app top bar doesn't reflect the current state 
- Bugfix [owncloud/web#11186](https://github.com/owncloud/web/pull/11186): Images stretched in preview app in Safari browser 
- Bugfix [owncloud/web#11194](https://github.com/owncloud/web/pull/11194): UI breaks when tags are numbers 
- Bugfix [owncloud/web#11253](https://github.com/owncloud/web/pull/11253): Open dropdown menu does not deselect other items in admin settings app 
- Change [owncloud/web#11251](https://github.com/owncloud/web/pull/11251): Removal of Deprecated Config Options 
- Change [owncloud/web#11252](https://github.com/owncloud/web/pull/11252): Remove draw-io as default app 
- Change [owncloud/web#11277](https://github.com/owncloud/web/pull/11277): Remove set as description space action 
- Enhancement [owncloud/web#11166](https://github.com/owncloud/web/pull/11166): Add share role icon to shared with me table
- Enhancement [owncloud/web#11258](https://github.com/owncloud/web/pull/11258): Application menu extension point 
- Enhancement [owncloud/web#11279](https://github.com/owncloud/web/pull/11279): Move quota info to general info in user menu 
- Enhancement [owncloud/web#11280](https://github.com/owncloud/web/pull/11280): Add edit description button to space info

### Reva

*   Bugfix [cs3org/reva#4741](https://github.com/cs3org/reva/pull/4741): Always find unique providers
*   Bugfix [cs3org/reva#4762](https://github.com/cs3org/reva/pull/4762): Blanks in dav Content-Disposition header
*   Bugfix [cs3org/reva#4775](https://github.com/cs3org/reva/pull/4775): Fixed the response code when copying the shared from to personal
*   Bugfix [cs3org/reva#4633](https://github.com/cs3org/reva/pull/4633): Allow all users to create internal links
*   Bugfix [cs3org/reva#4771](https://github.com/cs3org/reva/pull/4771): Deleting resources via their id
*   Bugfix [cs3org/reva#4768](https://github.com/cs3org/reva/pull/4768): Fixed the file name validation if nodeid is used
*   Bugfix [cs3org/reva#4758](https://github.com/cs3org/reva/pull/4758): Fix moving locked files, enable handling locked files via ocdav
*   Bugfix [cs3org/reva#4774](https://github.com/cs3org/reva/pull/4774): Fix micro ocdav service init and registration
*   Bugfix [cs3org/reva#4776](https://github.com/cs3org/reva/pull/4776): Fix response code for DEL file that in postprocessing
*   Bugfix [cs3org/reva#4746](https://github.com/cs3org/reva/pull/4746): Uploading the same file multiple times leads to orphaned blobs
*   Bugfix [cs3org/reva#4778](https://github.com/cs3org/reva/pull/4778): Zero byte uploads
*   Change [cs3org/reva#4759](https://github.com/cs3org/reva/pull/4759): Updated to the latest version of the go-cs3apis
*   Change [cs3org/reva#4773](https://github.com/cs3org/reva/pull/4773): Ocis bumped
*   Enhancement [cs3org/reva#4766](https://github.com/cs3org/reva/pull/4766): Set archiver output format via query parameter
*   Enhancement [cs3org/reva#4763](https://github.com/cs3org/reva/pull/4763): Improve posixfs storage driver
